lib/pdf/collate.rb

```diff
@@ -75,6 +75,17 @@ def self.merge_local_pdfs(output_file, *filenames)
     end
   end
 
+  # PDFs generated by puppeteer and then combined using ghostscript tended to have
+  # missing fonts which would make them unreadable in some PDF viewers.  This issue
```

Comment:
I'd go as far as to specify here that puppeteer generates PDFs with embedded font names that can be the same between different PDFs, leading to problems when we try to merge the resulting PDFs. Whereas ghostscript adds a unique hash to each embedding, allowing for a seamless merge.

I also tracked down the SO answer that led us to this conclusion; probably worth linking https://stackoverflow.com/questions/52350486/rename-pdf-embedded-font/52352156#52352156